tests/dragonfly/cluster_test.py

```diff
@@ -1991,3 +1991,58 @@ async def test_replicate_disconnect_redis_cluster(redis_cluster, df_factory, df_
     await c_replica.execute_command("REPLICAOF NO ONE")
     capture = await seeder.capture()
     assert await seeder.compare(capture, replica.port)
+
+
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes"})
+async def test_cluster_memory_consumption_migration(df_factory: DflyInstanceFactory):
+    # Check data migration from one node to another
+    instances = [
+        df_factory.create(
+            port=BASE_PORT + i,
+            admin_port=BASE_PORT + i + 1000,
+            vmodule="outgoing_slot_migration=9,cluster_family=9,incoming_slot_migration=9",
+        )
+        for i in range(3)
+    ]
+
+    replica = df_factory.create(
+        port=BASE_PORT + 3,
+        admin_port=BASE_PORT + 3 + 1000,
+        vmodule="outgoing_slot_migration=9,cluster_family=9,incoming_slot_migration=9",
+    )
+
+    df_factory.start_all(instances)
+    df_factory.start_all([replica])
```

Comment:
just a general note you can concat instance + replica into one statement. No action needed here just echoing the info :smile: